warehouse/query-core/src/main/java/datawave/query/iterator/logic/TermFrequencyExcerptIterator.java

```diff
@@ -902,24 +948,24 @@ private static String getDtUid(String str) {
     }
 
     public void setHitTermsList(List<String> hitTermsList) {
-        this.hitTermsList = (ArrayList<String>) hitTermsList;
+        this.config.hitTermsList = (ArrayList<String>) hitTermsList;
```

Comment:
it's save to do call `this.config` without a null check here b/c the `config` instance is _always_ created and _final_ so it can't be replaced inadvertently with a null.